### DISCUSSION TITLE
Make the examples outlook compatible

### DISCUSSION THREAD

**Discussion by maclementED** [resend/react-email#620] [Ideas]
### Goals

The goal of this feature request is to ensure that all examples are rendered correctly on Outlook. Currently, there are some examples that do not display properly on this email client.

### Background

This is how the Airbnb review look likes on outlook
<img width="492" alt="image" src="https://user-images.githubusercontent.com/67877770/229865429-bfcfe27f-b517-430d-85c7-194d47cb4508.png">

### Proposal

To implement this feature, we would need to check the examples with Outlook to verify if they are compatible with Outlook's email formatting standards

**Comment by bukinoshita** [comment]
Nice, let's make this happen!
